### ISSUE TITLE
MPD: Unnecessary multiple idle subsystem events 

### ISSUE THREAD

**Issue by torumyax** [mopidy/mopidy#1662]

**Comment by torumyax** [comment]
I forgot to mention that I'm using Mopidy 2.1.0 from apt repo on Ubuntu 17.10. And sorry for my English. English isn't my language. Thanks!

**Comment by kingosticks** [comment]
Why closed? Is the report incorrect?

**Comment by torumyax** [comment]
Hello. I'm sorry. I just thought this wasn't much of an "issue". And I did not want to trouble you...so.

**Comment by kingosticks** [comment]
No, all issues are important and the simpler ones make good starters for people new to the project.

**Comment by torumyax** [comment]
OK. I'll try to write again. Again sorry for the inconvenience.

* If I know Python, I would contribute to the project, but ... all I can is to report. I did write an introduction to Mopidy in Japanese in my blog though.

**Comment by kingosticks** [comment]
Please, never edit posts and delete what you wrote on any forum unless you really have to.

**Comment by torumyax** [comment]
I'll try to keep it in mind. I'm glad that I didn't offend or anything at least.
